test/functional/sqlsrv/srv_699_out_param_integer.phpt

```diff
@@ -0,0 +1,94 @@
+--TEST--
+GitHub issue #699 - binding integer as output parameter failed 
+--DESCRIPTION--
+This test uses the sample stored procedure provided by the user, in which an 
+error situation caused the binding to fail with an irrelevant error message about UTF-8 translation. This test proves that this issue has been fixed.
+--ENV--
+PHPT_EXEC=true
+--SKIPIF--
+<?php require('skipif.inc'); ?>
+--FILE--
+<?php
+
+if (strtoupper(substr(PHP_OS, 0, 3)) === 'LIN') {

```

Comment:
Is it better to put this in a skipif?

Reply:
no, because this is supposed to be temporary, I hope :)

Reply:
I changed this part to add a workaround to make the test pass in Linux